script/testing/oltpbench/test_oltpbench.py

```diff
@@ -179,23 +180,23 @@ def validate_result(self):
         # If it's not there, we'll dump out the contents of the directory to make it
         # easier to determine whether or not we are crazy when running Jenkins.
         if not os.path.exists(self.test_histogram_path):
+            LOG.error("Unable to find OLTP-Bench result file '{}'".format(self.test_histogram_path))
             print("=" * 50)
```

Comment:
We might also want to replace Line 184 - 189 'print' to 'LOG.error()' because they contain useful information for debugging.